lib/benchee/conversion/count.ex

```diff
@@ -151,7 +161,7 @@ defmodule Benchee.Conversion.Count do
       iex> Benchee.Conversion.Count.format(45.6789)
       "45.68"
 
-      iex> Benchee.Conversion.Count.format({45.6789, :thousand})
+      iex> Benchee.Conversion.Count.format({45.6789, %Benchee.Conversion.Unit{long: "Thousand", magnitude: "1_000", short: "K"}})
```

Comment:
Reworking the API is not a problem, did it in another place just ups the spaces where we duplicate stuff but maybe we can get rid off that in another PR/place :D